README.md

```diff
@@ -1,2 +1,70 @@
-# resty
-An API testing tool that can be run in any browser
+# Getting Started with Create React App
```

Comment:
Rename this one to old-Readme.md(or whatever name you want) and make the other one README.md so that way github knows to post that one up instead.

Reply:
Updated the readme.